src/main.cpp

```diff
@@ -956,6 +959,10 @@ int main(int argc, char *argv[])
                 params.muxer = optarg;
                 break;
 
+            case 's':
+                params.show_mouse = 1;
```

Comment:
hey @Stronkkey just to check if I understand this patch correctly: currently `wf-recorder` records the mouse cursor (at least, I see it).

After this patch there should be a new option to ... _disable_ the mouse cursor or _enable_ it?

I'm slightly confused by the option name and the effect that it has :)

thanks

Reply:
Fixed now.